### DISCUSSION TITLE
How to a set a icon and title to the game window

### DISCUSSION THREAD

**Discussion by AbhayRog** [pokepetter/ursina#229] [Q&A]
How to a set a icon and title to the game window, Thanks!

**Comment by PratangsuRakshit** [comment]
For title you can - 
Window.borderless = True
Window.title = ''

**Reply by PratangsuRakshit** [reply]
Does it works

**Comment by rhackerd** [comment]
window.borderless = False
window.title = "here your title"

**Comment by C0rupted** [comment]
Make sure you insert this code **BEFORE** initialising the Ursina class:
```python
window.borderless = False
window.title = "your title here"
window.icon = "path/to/icon.ico"
```

**Reply by nashubashu** [reply]
After setting the icon properly this error shows up and icon doesn't work

**`display:windisplay(warning): windows icon filename 'C:\Users\D-Mit\Desktop\Devote\Windows-icon.png' could not be loaded!!`**

my code:-

```
window.borderless = False
window.title = 'Devote'
window.icon = 'Devote.png'
app = Ursina()
```

**Reply by pokepetter** [reply]
Icons should be .ico, not .png.

**Comment by Carlisle03** [comment]
> How to a set a icon and title to the game window, Thanks!


To set an icon and title to the game window,
Do this.

```python
from ursina import *
app = Ursina(icon="C:\path\to\icon.png", title=None)
#                                                                                ⬆ Remove none and replace it as your custom icon
app.run()
```

This should work.
replace 'C:\path\to\icon.png' into your path to the icon
REMAINDER: The icon can be either .ico or .png(.png worked on macOS)

I hope this helps.
